build.sbt

```diff
@@ -1,14 +1,15 @@
 import Dependencies._
+import sbt.Keys.scalacOptions
 
 inThisBuild(
   List(
     onLoadMessage := s"Welcome to scalafix ${version.value}",
     scalaVersion := scala213,
     crossScalaVersions := List(scala213, scala212, scala211),
     fork := true,
+    scalacOptions ++= syntheticsOn.value,
```

Comment:
I guess you can now revert this change and get rid of `syntheticsOn` since the scala3 customization is only applied where scala3 cross-building is enabled.